src/middlewared/middlewared/plugins/virt/instance.py

```diff
@@ -274,6 +277,9 @@ def __data_to_config(self, instance_name: str, data: dict, raw: dict = None, ins
             config['user.autostart'] = str(data['autostart']).lower()
 
         if instance_type == 'VM':
+            if data.get('image_os'):
+                config['image.os'] = data['image_os'].capitalize()
```

Comment:
Is this clearly defined somewhere in incus specs/docs that we can use it this way? Should we be using `user.ix_image_os` instead?

Reply:
It is documented here https://github.com/lxc/incus/pull/1767/files

We are investigating upgrading incus version as well and that should hopefully pull this fix in too when that happens

Reply:
Specifically this part 
```
When creating a Windows virtual machine, make sure to set the `image.os` property to something starting with `Windows`.
Doing so will tell Incus to expect Windows to be running inside of the virtual machine and to tweak behavior accordingly.
```